roofit/roofitcore/inc/RooParamBinning.h

```diff
@@ -70,7 +70,7 @@ class RooParamBinning : public RooAbsBinning {
   RooAbsReal* xlo() const { return _lp ? ((RooAbsReal*)_lp->at(0)) : _xlo ; }
   RooAbsReal* xhi() const { return _lp ? ((RooAbsReal*)_lp->at(1)) : _xhi ; }
 
-  ClassDef(RooParamBinning,2) // Binning specification with ranges parameterized by external RooAbsReal functions
+  ClassDef(RooParamBinning,3) // Binning specification with ranges parameterized by external RooAbsReal functions
```

Comment:
This shouldn't be necessary if the typedefs would be done correctly. Double check them:
the warning message hints at a pointer-vs-object difference.

Reply:
I triple-checked this, and there is no inconsistency with the `typedefs`. I suppose it's some quirk in IO that the warning appears.